src/main/java/cat/udl/eps/entsoftarch/repository/ClosedLicenseRepository.java

```diff
@@ -0,0 +1,17 @@
+package cat.udl.eps.entsoftarch.repository;
+
+import cat.udl.eps.entsoftarch.domain.ClosedLicense;
+
+import org.springframework.data.repository.PagingAndSortingRepository;
+import org.springframework.data.repository.query.Param;
+import org.springframework.data.rest.core.annotation.RepositoryRestResource;
+
+import java.util.List;
+
+/**
+ * Created by Victor on 03/03/2017.
+ */
+@RepositoryRestResource(collectionResourceRel="closedlicenses", path="closedlicenses")
```

Comment:
Better to keep the default collectionResourceRel "closedLicense" to follow the same approach through the whole API